test/squish/suite_VerifySettingsDialog/shared/scripts/gui_helpers.py

```diff
@@ -0,0 +1,181 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# -*- coding: utf-8 -*
+# mypy: disable-error-code="attr-defined"
+
+import gui_locators
+import squish
+import test
+
+snoozeTimeout = 3  # seconds
```

Comment:
Do you have an example? (Is this comment referring to lines 6-9?) 
I ran linters, etc, but there were issues since some of the Python libraries are named the same as Squish libraries and we need to point correctly to the Squish libraries (hence adding `# mypy: disable-error-code="attr-defined"`).